services/s3/config/crd/bases/s3.services.k8s.aws_buckets.yaml

```diff
@@ -15,122 +15,122 @@ spec:
+                      strategy for resources that do not have ARN in CreateOutputResponse
+                      https://github.com/aws/aws-controllers-k8s/issues/270'
                     type: string
-                  type:
-                    description: Type is the type of the Condition
+                  ownerAccountID:
+                    description: OwnerAccountID is the AWS Account ID of the account
+                      that owns the backend AWS service API resource.
                     type: string
                 required:
-                - status
-                - type
+                - ownerAccountID
                 type: object
-              type: array
-            location:
-              type: string
-          required:
-          - ackResourceMetadata
-          - conditions
-          type: object
-      type: object
-  version: v1alpha1
-  versions:
-  - name: v1alpha1
+              conditions:
+                description: All CRS managed by ACK have a common `Status.Conditions`
+                  member that contains a collection of `ackv1alpha1.Condition` objects
+                  that describe the various terminal states of the CR and its backend
+                  AWS service API resource
+                items:
+                  description: Condition is the common struct used by all CRDs managed
+                    by ACK service controllers to indicate terminal states  of the
+                    CR and its backend AWS service API resource
+                  properties:
+                    lastTransitionTime:
+                      description: Last time the condition transitioned from one status
+                        to another.
+                      format: date-time
+                      type: string
+                    message:
+                      description: A human readable message indicating details about
+                        the transition.
+                      type: string
+                    reason:
+                      description: The reason for the condition's last transition.
+                      type: string
+                    status:
+                      description: Status of the condition, one of True, False, Unknown.
+                      type: string
+                    type:
+                      description: Type is the type of the Condition
+                      type: string
+                  required:
+                  - status
+                  - type
+                  type: object
+                type: array
+              location:
+                type: string
+            required:
+            - ackResourceMetadata
+            - conditions
+            type: object
+        type: object
     served: true
     storage: true
+    subresources:
+      status: {}
 status:
   acceptedNames:
     kind: ""
     plural: ""
-  conditions: null
-  storedVersions: null
+  conditions: []
+  storedVersions: []
```

Comment:
all these changes to CRD YAML files should be in a separate PR. If these changes were from an updated kubebuilder, then we should have a separate PR that only updates the service controller generated files due to that upgraded KB version.

Reply:
Yes this change was made by the latest version of `controller-gen`